package.json

```diff
@@ -158,6 +158,7 @@
     "jest-fetch-mock": "^3.0.3",
     "jest-mock-extended": "^3.0.1",
     "next-router-mock": "^0.9.1-beta.0",
+    "node-gyp": "^10.0.1",
```

Comment:
Is this used somewhere?

Reply:
ah, no... was used for trouble shooting earlier.  Forgot to remove.